purchases_ui_flutter/lib/purchases_ui_flutter.dart

```diff
@@ -49,6 +49,13 @@ class RevenueCatUI {
     return _parseStringToResult(result);
   }
 
+  static Future<void> presentCustomerCenter() async {
+    // handling result will be implemented in upcoming PRs
```

Comment:
I actually don't think we need to handle anything, since the actions will be sent as a callback not as a result